integration-webcert/src/main/java/se/inera/intyg/minaintyg/integration/webcert/converter/data/value/IcfValueConverter.java

```diff
@@ -17,14 +18,23 @@ public CertificateDataValueType getType() {
 
   @Override
   public CertificateQuestionValue convertToValue(CertificateDataElement element) {
-    final var value = ((CertificateDataIcfValue) element.getValue()).getText();
-    // TODO: Handle icf-codes
-    return getText(value);
+    final var icfCodes = ((CertificateDataIcfValue) element.getValue()).getIcfCodes();
+    final var text = ((CertificateDataIcfValue) element.getValue()).getText();
```

Comment:
Take this a suggestion, but would it look cleaner if we instead casted the value once, and then used that variable? Something like this:

```java
final var value = (CertificateDataIcfValue) element.getValue;
```
And then use value to check icfCodes or text?